ts/common.ts

```diff
@@ -1,24 +1,15 @@
 export const networkSpecificConfigs = {
   mainnet: {
     // NOTE: replace with deployed address
-    module: '0x90E75f390332356426B60FB440DF23f860F6A113',
+    module: '0x4c04377f90Eb1E42D845AB21De874803B8773669',
     rpcUrl: 'https://eth.llamarpc.com',
     gpv2Settlement: '0x9008D19f58AAbD9eD0D60971565AA8510560ab41',
```

Comment:
FYI, I think for this, even the `gpv2settlement` contract should be specified as an immutable value within the module, so this can be pulled from the module as well. This does make deployment of the module slightly more expensive, but also makes it very easy to be able to validate the setup of the module from public functions (as well as makes it so that all we have to specify when running this is the module's deployment address and the RPC).